### PR TITLE
Multiple vulnerabilities in RubyGems

### PR DESCRIPTION
**Pull request by unak**

### CODE REVIEW

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+There are multiple vulnerabilites in RubyGems bundled by Ruby.
+It is [reported at the official blog of RubyGems](http://blog.rubygems.org/2017/08/27/2.6.13-released.html).
+
+## Details
+
+The following vulnerabilites have been reported.
+
+* a DNS request hijacking vulnerability
+* an ANSI escape sequence vulnerability
+* a DOS vulernerability in the query command
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: DoS

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* a vulnerability in the gem installer that allowed a malicious gem to overwrite arbitrary files
+
+It is strongly recommended for Ruby users to take one of the following workarounds as soon as possible.
+
+## Affected Versions
+
+* Ruby 2.2.7 and all versions of Ruby 2.2 prior to it
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: "2.2.0 - 2.2.7" is enough? "and all versions of Ruby 2.x" looks too verbose to me.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* Ruby 2.3.4 and all versions of Ruby 2.3 prior to it
+* Ruby 2.4.1 and all versions of Ruby 2.4 prior to it
+* prior to trunk revision 59672
+
+## Workarounds
+
+At this time, the fixed versions of Ruby have not been released yet.
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: > At this moment, there are no Ruby releases including the fix for RubyGems.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+But you can upgrade RubyGems to latest version (2.6.13 or later), then the vulnerabilites will be fixed.
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: > but you can upgrade RubyGems to the latest version. RubyGems 2.6.13 or later includes the fix for the vulnerabilities.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+gem update --system
+```
+
+If you can not upgrade RubyGems, the following patches can be applied as workarounds.
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: > If you can't upgrade RubyGems, You can apply the following patches as a workaround.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+But you can upgrade RubyGems to the latest version.
+RubyGems 2.6.13 or later includes the fix for the vulnerabilities.
+
+
+```
+gem update --system
+```
+
+If you can't upgrade RubyGems, you can apply the following patches as a workaround.
+For Ruby 2.4.1, apply the two patches one by one.
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: For Ruby 2.4.1, you'll need 2 patches.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+* [for Ruby 2.2.7](https://bugs.ruby-lang.org/attachments/download/6690/rubygems-2613-ruby22.patch)
+* [for Ruby 2.3.4](https://bugs.ruby-lang.org/attachments/download/6691/rubygems-2613-ruby23.patch)
+* [for Ruby 2.4.1 (RubyGems 2.6.11 to 2.6.12)](https://bugs.ruby-lang.org/attachments/download/6692/rubygems-2612-ruby24.patch)
+* [for Ruby 2.4.1 (RubyGems 2.6.12 to 2.6.13)](https://bugs.ruby-lang.org/attachments/download/6693/rubygems-2613-ruby24.patch)
+
+About trunk, sync the latest revision.
```
Comment (sorah) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: About the trunk, update to the latest revision.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+layout: news_post
+title: "Multiple vulnerabilities in RubyGems"
+author: "usa"
+translator:
+date: 2017-08-29 12:00:00 +0000
+tags: security
+lang: en
+---
+
+There are multiple vulnerabilites in RubyGems bundled by Ruby.
```
Comment (koic) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: It looks like a typo of `vulnerabilities`. There is also the same typo on L16.

File: en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+gem update --system
+```
+
+If you can't upgrade RubyGems, you can apply the following patches as a workaround.
+
+* [for Ruby 2.2.7](https://bugs.ruby-lang.org/attachments/download/6690/rubygems-2613-ruby22.patch)
+* [for Ruby 2.3.4](https://bugs.ruby-lang.org/attachments/download/6691/rubygems-2613-ruby23.patch)
+* for Ruby 2.4.1: need 2 patches.  Apply sequentially as follows:
```
Comment (meisyal) [inline en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: I think there is an extra space before `Apply` word. Should it be removed?

**Pull request merged**